Index: src/examples/TestServer.java
===================================================================
RCS file: /home/cvs/postgis/postgis/jdbc2/src/examples/TestServer.java,v
retrieving revision 1.3
diff -U2 -r1.3 TestServer.java
--- src/examples/TestServer.java	4 Feb 2005 09:05:24 -0000	1.3
+++ src/examples/TestServer.java	8 Feb 2005 16:50:15 -0000
@@ -64,6 +64,6 @@
             conn = DriverManager.getConnection(dburl, dbuser, dbpass);
             System.out.println("Adding geometric type entries...");
-            ((org.postgresql.PGConnection) conn).addDataType("geometry", "org.postgis.PGgeometry");
-            ((org.postgresql.PGConnection) conn).addDataType("box3d", "org.postgis.PGbox3d");
+            ((org.postgresql.Connection) conn).addDataType("geometry", "org.postgis.PGgeometry");
+            ((org.postgresql.Connection) conn).addDataType("box3d", "org.postgis.PGbox3d");
             Statement s = conn.createStatement();
             System.out.println("Creating table with geometric types...");
Index: src/org/postgis/DriverWrapper.java
===================================================================
RCS file: /home/cvs/postgis/postgis/jdbc2/src/org/postgis/DriverWrapper.java,v
retrieving revision 1.5
diff -U2 -r1.5 DriverWrapper.java
--- src/org/postgis/DriverWrapper.java	4 Feb 2005 09:05:24 -0000	1.5
+++ src/org/postgis/DriverWrapper.java	8 Feb 2005 16:50:15 -0000
@@ -26,5 +26,4 @@
 
 import org.postgresql.Driver;
-import org.postgresql.PGConnection;
 
 import java.sql.Connection;
@@ -69,5 +68,5 @@
     public static final String REVISION = "$Revision: 1.5 $";
 
-    public DriverWrapper() {
+    public DriverWrapper() throws java.sql.SQLException {
         super();
     }
@@ -97,5 +96,5 @@
         url = mangleURL(url);
         Connection result = super.connect(url, info);
-        addGISTypes((PGConnection) result);
+        addGISTypes((org.postgresql.Connection) result);
         return result;
     }
@@ -104,5 +103,5 @@
      * adds the JTS/PostGIS Data types to a PG Connection.
      */
-    public static void addGISTypes(PGConnection pgconn) {
+    public static void addGISTypes(org.postgresql.Connection pgconn) {
         // This is correct for PostgreSQL jdbc drivers up to V7.4
         pgconn.addDataType("geometry", "org.postgis.PGgeometry");
